docs/admin/access.rst

```diff
@@ -80,82 +78,81 @@ The default value can be changed by :setting:`DEFAULT_ACCESS_CONTROL`.
 Managing per-project access control
 +++++++++++++++++++++++++++++++++++
 
-Users with the :guilabel:`Manage project access` privilege (see
-:ref:`privileges`) can manage users in projects with non-`Custom` access
-control. They can assign users to one of the following groups.
+Users granted :guilabel:`Manage project access` (see :ref:`privileges`)
+can manage users in all non-`Custom` projects by assigning users
+to one of the following groups:
 
 For `Public`, `Protected` and `Private` projects:
 
+Granting users :guilabel:`Manage project access` (see :ref:`privileges`)
+gives them access to assign other users in Public`, `Protected` and
+`Private` (but not `Custom`) projects to one of the following groups:
+
 Administration
-    Includes all permissions available for the project.
+    All available permissions for the project.
```

Comment:
Including adding people that can add administrators?
Should this be site-wide administration? What I am asking is if "superuser" is above this?

Reply:
This is only at project level - it grants all permissions for the project.

Reply:
Yes, instance superuser is above this and doesn’t need to be mentioned here